### DISCUSSION TITLE
FCM 'voip' notifications

### DISCUSSION THREAD

**Discussion by er1cAk** [invertase/react-native-firebase#5420] [Q&A]
Am I able to send voip notification?

**Comment by mikehardy** [comment]
You may send firebase cloud messages according to the related firebase messaging / firebase cloud messaging documentation.

**Reply by er1cAk** [reply]
so FCM does not support voip notifications

**Reply by mikehardy** [reply]
That is not what I said. I said:

> You may send firebase cloud messages according to the related firebase messaging / firebase cloud messaging documentation.

You did not specify what a "voip notification" is. I imagine you may send whatever payload you want.
Then to handle it you'll integrate with some sort of notification package and use that do a full screen notification or similar. There are cases of Notifee users trying these but I'm not sure people have achieved success yet: https://github.com/notifee/react-native-notifee/issues?q=voip - perhaps other packages work correctly.

Keep in mind: FCM === cloud messages sending JSON to devices, "Local Notifications" === some local notifications package doing things visible to users. They're separate, if related.
